Database/Corrections/sodQuestFixes.lua

```diff
@@ -3312,6 +3312,42 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410097,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90219] = {
+            [questKeys.name] = "Shield Mastery",
+            [questKeys.startedBy] = {nil,{450011}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 5,
+            [questKeys.requiredRaces] = raceIDs.ORC,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Find a Galvanic Icon, loot it and equip the Icon, follow its instructions to kill 10 enemies with Lightning Bolt Rank 1 to receive the rune."},
+            [questKeys.requiredSpell] = -410098,
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
+        [90220] = {
+            [questKeys.name] = "Shield Mastery",
+            [questKeys.startedBy] = {nil,{450011}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 5,
+            [questKeys.requiredRaces] = raceIDs.TROLL,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Find a Galvanic Icon, loot it and equip the Icon, follow its instructions to kill 10 enemies with Lightning Bolt Rank 1 to receive the rune."},
+            [questKeys.requiredSpell] = -410098,
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
```

Comment:
You might want to combine these two quests, as they are absolutely the same for Orc and Troll. To do this all you need to do is to change `requiredRaces` to:

```lua
[questKeys.requiredRaces] = raceIDs.ORC + raceIDs.TROLL,
```

If you combine them make sure to also remove the leftover ID in `SeasonOfDiscovery.lua`

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5612/commits/dc1eb840eb72214bd6bff48a352c243960e6e840